pkg/proxy/snapshot.go

```diff
@@ -135,7 +135,24 @@ func (p *Proxy) SnapshotPurgeStatus(ctx context.Context, req *rpc.ProxyEngineReq
 		Status: map[string]*eptypes.SnapshotPurgeStatusResponse{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.